index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="style.css" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books</title>
+</head>
+<body>
+    <div>
+        <h1>Awesome Books</h1>
+      </div>
+      <div id="show-book">
+        <ul id="book-list">
+        </ul>
+      </div>
+    <form class="form">                                        
+        <input type="text" placeholder="Title" class="title"/> <br> 
+        <input type="text" placeholder="Author" class="author"/>            
+    </form>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add a feature that resets your `inputs` after adding a books to prevent a user to add the same book many times. To help you to do, you can refer this **[article](https://www.w3schools.com/jsref/met_form_reset.asp)**.